index.js

```diff
@@ -1,10 +1,19 @@
 'use strict';
 const {URL} = require('url');
+const HttpsAgent = require('https').Agent;
+const HttpAgent = require('http').Agent;
 const got = require('got');
 const registryUrl = require('registry-url');
 const registryAuthToken = require('registry-auth-token');
 const semver = require('semver');
 
+const agentOptions = {
+	keepAlive: true,
+	maxSockets: 50
```

Comment:
Can you add a code comment on why this number was chosen?

Reply:
Added a short sentence and references to the npm client / PR accordingly.